scorer/verifications/seed_connected.py

```diff
@@ -56,17 +59,17 @@ def verify(block):
         for c in connections:
             u = c['_to'].replace('users/', '')
             if u not in users:
-                users[u] = {'connected': [], 'reported': []}
+                users[u] = {'connected': [], 'reported': [], 'communities': []}
 
             if c['level'] in ['just met', 'already known', 'recovery']:
-                already_connected = seed_group['_key'] in users[u]['connected']
-                if not already_connected:
+                if seed_group['_key'] not in users[u]['communities']:
+                    users[u]['communities'].append(seed_group['_key'])
+                if seed_group['_key'] not in users[u]['connected']:
```

Comment:
If I understand this correctly, a "community" is a seed group with 0 quota.  I think this is a simple implementation that achieves our goals.

Should a new community be added as a "seed group" with a vote from seed dao but with 0 quota?

Reply:
yes, if we want those groups to be known differently globally, we have no way other than assigning the attribute that makes them different (seed with 0 quota) by something like seed dao to them.